e2e/metrics_test.go

```diff
@@ -75,38 +74,57 @@ var _ = Describe("Metrics functional tests", func() {
 
 		When("Some query results are cached", func() {
 			BeforeEach(func() {
-				Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_entry_count 0"))
-				Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_hit_count 0"))
-				Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_miss_count 0"))
+				Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(
+					SatisfyAll(
+						ContainElement("blocky_cache_entry_count 0"),
+						ContainElement("blocky_cache_hit_count 0"),
+						ContainElement("blocky_cache_miss_count 0"),
+					))
 			})
 
 			It("Should increment cache counts", func() {
-				msg := util.NewMsgWithQuestion("google.de.", dns.Type(dns.TypeA))
+				msg := util.NewMsgWithQuestion("google.de.", A)
 
 				By("first query, should increment the cache miss count and the total count", func() {
-					Expect(doDNSRequest(blocky, msg)).Should(BeDNSRecord("google.de.", dns.TypeA, 123, "1.2.3.4"))
-
-					Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_entry_count 1"))
-					Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_hit_count 0"))
-					Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(ContainElement("blocky_cache_miss_count 1"))
+					Expect(doDNSRequest(blocky, msg)).
+						Should(SatisfyAll(
+							BeDNSRecord("google.de.", A, "1.2.3.4"),
+							HaveTTL(BeNumerically("==", 123)),
+						))
+
+					Eventually(fetchBlockyMetrics).WithArguments(metricsURL).Should(
+						SatisfyAll(
```

Comment:
I think using one indentation style for `Should` `SatisfyAll` would benefit the readability of the tests.
Currently there are:
```Go
.Should(
  SatisfyAll(
```
```Go
.Should(SatisfyAll(
```
```Go
.
Should(
  SatisfyAll(
```

Reply:
I agree with this in principle. We probably have other examples in the code that should be reformatted along the same lines. I would like to "fix" it in some automated way. I have tried the tool "[golines](https://github.com/segmentio/golines)". It breaks long lines, but in this particular case the line is too short. Do you have any idea how to automate this?

Reply:
I haven't looked at the docs to try and figure out if it can do this, but [gofumpt](https://github.com/mvdan/gofumpt) is "a stricter gofmt", and might have a rule that applies.

Reply:
Skimmed over it and it seems it won't solve this particular problem but looks like a great tool for readability enhancement.

Reply:
Tested gofump and unfortunately it breaks the "only one cuddle assignment allowed before if statement" linter rule.
I tinker around with it for a bit.

Reply:
I added linebreaks on "SatisfyAll" manually